src/scss/_sofie-tooltip-hover.scss

```diff
@@ -1,4 +1,4 @@
-.c-hover-scrub-hover-element {
+.c-tooltip-hover-element {
```

Comment:
Since you are changing the name of this class anyway, you might as well go and remove the BEMIT convetion, since we don't want to use that.

Reply:
I removed the BEMIT convention from this class.